### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -12918,26 +12919,35 @@ settting->setting
...
 sh1sum->sha1sum
 shadasloo->shadaloo
 shaddow->shadow
 shadhow->shadow
 shadoloo->shadaloo
 shamen->shaman, shamans,
+shandeleer->chandelier
 shandow->shadow
+shaneal->chenille
+shanghi->shanghai
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Capitalise the suggestion.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -12918,26 +12919,35 @@ settting->setting
...
 sharable->shareable
+sharloton->charlatan
+sharraids->charades
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the singular.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -15356,6 +15356,7 @@ sesssions->sessions
 setgit->setgid
 seting->setting
 setings->settings
+setis->set is
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest settees?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -15369,6 +15370,7 @@ settting->setting
 setttings->settings
 seuence->sequence
 seuences->sequences
+sevaral->several
 seve->save
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Not one of yours but suggest sieve too.

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -15379,22 +15381,31 @@ severitie->severity, severities,
 severley->severely
 severly->severely
 sevice->service
+sewdonim->pseudonym
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the plural

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -15379,22 +15381,31 @@ severitie->severity, severities,
...
 sgadow->shadow
 sh1sum->sha1sum
 shadasloo->shadaloo
 shaddow->shadow
 shadhow->shadow
 shadoloo->shadaloo
 shamen->shaman, shamans,
+shandeleer->chandelier
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the plural

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -15379,22 +15381,31 @@ severitie->severity, severities,
...
 shandow->shadow
+shaneal->chenille
+shanghi->Shanghai
 sharable->shareable
+sharloton->charlatan
+sharraid->charade
+sharraids->charades
 shashes->slashes
+shatow->chateaux
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: This is the plural, the singular is château (apparently the plural can be x or s).

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -15379,22 +15381,31 @@ severitie->severity, severities,
...
 shbang->shebang
 sheat->sheath, sheet, cheat,
 shedule->schedule
 sheduled->scheduled
 shedules->schedules
 sheduling->scheduling
+sheepherd->shepherd
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the plural

**Comment by peternewman** [comment]
There seems to be overlap between this PR and another, see e.g. https://github.com/codespell-project/codespell/pull/765#discussion_r287596819

File: codespell_lib/data/dictionary.txt (hunk 8 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 7 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Comment by Gelma** [comment]
> There seems to be overlap between this PR and another, see e.g. [#765 (comment)](https://github.com/codespell-project/codespell/pull/765#discussion_r287596819)

Ok. Not a big deal. Anyway, I'm going to rebase each PR merge by merge.

**Comment by peternewman** [comment]
> > There seems to be overlap between this PR and another, see e.g. [#765 (comment)](https://github.com/codespell-project/codespell/pull/765#discussion_r287596819)
> 
> Ok. Not a big deal. Anyway, I'm going to rebase each PR merge by merge.

Although I'd rather avoid duplicate reviewing everything. I assume they've just come from different sources rather than accidentally duplicating PRs?

**Comment by Gelma** [comment]
> Although I'd rather avoid duplicate reviewing everything. I assume they've just come from different sources rather than accidentally duplicating PRs?

My sources are de-duplicated  at the beginning. Probably same word from different sources!

**Pull request merged**